### PR TITLE
Allow for adding content to footer of Modal component

### PR DESCRIPTION
**Pull request by cerinman**
This PR adds a new prop to the Modal component to allow for passing in jsx for populating the footer with content.  This was done after a need to allow text links in the footer.

New Prop:

footerContent: type(node) - used to add jsx to footer of the component.

@mxenabled/frontend-engineers 

Large Modal

With tool tip:
screen shot 2016-01-06 at 1 35 36 pm

With out tool tip:
screen shot 2016-01-06 at 1 36 03 pm

No footer content:
screen shot 2016-01-06 at 1 40 05 pm

Small Modal

With tool tip:
screen shot 2016-01-06 at 1 36 25 pm

With out tool tip:
screen shot 2016-01-06 at 1 36 42 pm

No footer content:
screen shot 2016-01-06 at 1 40 19 pm

### CODE REVIEW

File: src/components/Modal.js [hunk 1]
```diff
@@ -73,7 +82,7 @@ class Modal extends React.Component {
   _renderTooltipIconAndLabel () {
     if (this.props.tooltip) {
       return (
-        <div>
+        <div className='mx-modal-tooltip-label' style={styles.tooltipLabel}>
```
Comment (bsbeeks) [inline src/components/Modal.js]: Somewhat related...we don't need the else statement below. If there is no tooltip then just don't do anything in this method.

Comment (cerinman) [inline src/components/Modal.js]: will remove the else block

**Comment by bsbeeks** [comment]
after you remove the else block, this is good to :shipit:

**Comment by jmophoto** [comment]
:+1:

**Pull request merged**